### DISCUSSION TITLE
Failed to execute goal - Maven

### DISCUSSION THREAD

**Discussion by nubevi** [dds-jv/foro#204] [Consultas de herramientas] [answered]
Buenas tardes gente,

Estoy corriendo mvn verify y me tira esto:
La verdad que intente ver por mi cuenta qué era, pero ni caso.

![image](https://user-images.githubusercontent.com/26943847/116321090-d58ece80-a78f-11eb-9e72-088be2d8dc68.png)

**Comment by gastonprieto** [comment, marked as answer]
Buenas @nubevi 

Te esta diciendo que falla (validar cobertura), hiciste los tests? te fijate si mas arriba no te da un detalle mas explicito del problema ?

**Reply by nanderson87** [reply]
Más arriba te tiene que decir algo del estilo:

```
INFO] --- jacoco-maven-plugin:0.8.2:check (Validar Cobertura) @ ejercicio ---
[INFO] Loading execution data file {path}
[INFO] Analyzed bundle 'ejercicio' with 1 classes
[WARNING] Rule violated for package {package}: lines covered ratio is {number}, but expected minimum is 0.60
```

Donde _number_ va a ser el número de cobertura que tiene actualmente la aplicación. Para superar la verificación de jacoco la cobertura de la aplicación tiene que ser mayor a el 60%

**Reply by nubevi** [reply]
Si, era la falta de Tests. Gracias!
